rules/scala/BUILD

```diff
@@ -147,6 +147,7 @@ scalac_binary(
 annex_scala_binary(
     name = "test",
     srcs = glob(["test/**/*.scala"]),
+    main_class = "annex.TestRunner",
```

Comment:
@andyscott wanted to have automatic main class detection.

I'm ambivalent, but I imagine we should be consistent. Currently, the main class is written to a file, and then the file must be read.

Can be addressed later though.